src/api/useGetTodoList.tsx

```diff
@@ -1,17 +1,29 @@
 import { useQuery } from '@tanstack/react-query';
 import { getDb } from './mockDb';
+import { sleep } from '../utils/sleep';
+import { useRef } from 'react';
 
 export const todoListQueryKeys = {
   all: ['todoList'],
 };
 
-const getTodoList = async () => {
+const getTodoList = async ({ initialFetch }: { initialFetch: boolean }) => {
+  const sleepTime = initialFetch ? 2000 : 300;
```

Comment:
Please set it to:
```suggestion
  const sleepTime = initialFetch ? 2000 : 600;
```
these are the best values I think in delay to keep the feeling of the API